packages/core/src/core.ts

```diff
@@ -99,6 +99,7 @@ export function getInput(name: string, options?: InputOptions): string {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function setOutput(name: string, value: any): void {
+  process.stdout.write(os.EOL)
```

Comment:
i dont think it's necessary to add to all... adding to setOutput since it's more critical from a security perspective